### DISCUSSION TITLE
UI change

### DISCUSSION THREAD

**Discussion by bycolour2** [verdaccio/verdaccio#3767] [Q&A] [answered]
**First:**

I want to set up cache proxy with approve system. How could I achieve this?
Actually I want to create a buffer zone. Flow would look like:

npm i --registry ... -> Verdaccio download package to the store -> approve incoming package on the UI page -> if it's approved users can download it from local Verdaccio repository (cached package) 

**Second:**

I want to change UI, add some features for local use. How could I do that?
Maybe it's possible to use Verdaccio as API endpoints to my own web UI?

**Comment by juanpicado** [comment, marked as answer]
> Maybe it's possible to use Verdaccio as API endpoints to my own web UI?

You might, I wrote this while ago maybe helps https://verdaccio.org/docs/plugin-theme
